<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1527</width>
    <height>650</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SoundFFT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="rootLayout" stretch="0,1,0">
     <item>
      <widget class="QQuickWidget" name="quickWidget_deviceControllers">
       <property name="minimumSize">
        <size>
         <width>1280</width>
         <height>65</height>
        </size>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="0,1">
         <item row="1" column="1">
          <widget class="QQuickWidget" name="quickWidget_horizontal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>13</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QQuickWidget" name="quickWidget_vertical">
           <property name="minimumSize">
            <size>
             <width>13</width>
             <height>0</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="MyOpenGLWidget" name="glwidget" native="true">
           <property name="minimumSize">
            <size>
             <width>1024</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_charts">
           <item row="0" column="0">
            <widget class="MyOpenGLFreqGraphWidget" name="graph1_originalWave"/>
           </item>
           <item row="1" column="0">
            <widget class="MyOpenGLFreqGraphWidget" name="graph3_filteredWave"/>
           </item>
           <item row="0" column="1">
            <widget class="MyOpenGLFreqGraphWidget" name="graph2_originalFrequency"/>
           </item>
           <item row="1" column="1">
            <widget class="MyOpenGLFreqGraphWidget" name="graph4_filteredFrequency"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QQuickWidget" name="quickWidget_filterProfile">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>220</height>
              </size>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::SizeRootObjectToView</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QQuickWidget" name="quickWidget_filterListView">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::SizeRootObjectToView</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QQuickWidget" name="quickWidget_soundManager">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyOpenGLFreqGraphWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myopenglfreqgraphwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
